### DISCUSSION TITLE
Is SMT32 core is or will be added to PlatformIO?

### DISCUSSION THREAD

**Discussion by NitrofMtl** [stm32duino/Arduino_Core_STM32#1943] [Q&A] [answered]
Hi.

I added the STM32 core to arduino IDE, but I prefer using PIO.

But if I search for STM32 in new project, there's not the board chip I want (STM32G070CB).

In to the core file , ..2.4.0/tools, there's a PlatformIO folder. Is there a way manualy to add the core to PIO ?? 

Or is it on plan to add it shortly ?

Thanks.

Nitrof

**Comment by fpistm** [comment]
See https://github.com/platformio/platform-ststm32/issues/659

**Comment by NitrofMtl** [comment, marked as answer]
@fpistm  I donot understand how to use the work around...

**Reply by fpistm** [reply]
We do not support PIO nor integrate it. The script you mentioned was added by pio maintainer to integrate it. That's why I redirect you to the correct repo ro ask support 😉 
As I do not use PUI , I can't help on the workaround proposed.

**Reply by Miceuz** [reply]
@NitrofMtl just add 
```
platform_packages =
   framework-arduinoststm32@https://github.com/stm32duino/Arduino_Core_STM32.git#2.4.0
```
to your `platformio.ini` file and Platformio will pull in the 2.4.0 core.

**Reply by fpistm** [reply]
Thanks @Miceuz 
for the support 😉
